app/src/main/java/com/codesoom/assignment/springHandler.java

```diff
@@ -0,0 +1,21 @@
+package com.codesoom.assignment;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class springHandler implements HttpHandler {
```

Comment:
Java의 클래스는 관례적으로 첫 글자를 대문자로 씁니다. 클래스 이름을 다음과 같이 수정해주세요.
```suggestion
public class SpringHandler implements HttpHandler {
```

Reply:
앗! 수정하겠습니다